### ISSUE TITLE
SmartCity-Accessibility #310 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#197]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 81 lines...]
  location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[115,23] cannot find symbol
  symbol:   class OptionsWindow
  location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[74,25] cannot find symbol
  symbol:   class OptionsWindow
  location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality.ExtendedMapView
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[100,46] cannot find symbol
  symbol:   class OptionsWindow
  location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality.ExtendedMapView
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[116,28] cannot find symbol
  symbol:   class OptionsWindow
  location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality
[INFO] 6 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 14.242 s
[INFO] Finished at: 2017-01-19T16:26:00+00:00
[INFO] Final Memory: 28M/145M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure: Compilation failure:
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[49,46] cannot find symbol
[ERROR] symbol:   class OptionsWindow
[ERROR] location: package smartcity.accessibility.gui.components
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[111,43] cannot find symbol
[ERROR] symbol:   class OptionsWindow
[ERROR] location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[115,23] cannot find symbol
[ERROR] symbol:   class OptionsWindow
[ERROR] location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[74,25] cannot find symbol
[ERROR] symbol:   class OptionsWindow
[ERROR] location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality.ExtendedMapView
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[100,46] cannot find symbol
[ERROR] symbol:   class OptionsWindow
[ERROR] location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality.ExtendedMapView
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/JxMapsFunctionality.java:[116,28] cannot find symbol
[ERROR] symbol:   class OptionsWindow
[ERROR] location: class smartcity.accessibility.mapmanagement.JxMapsFunctionality
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] 200ca0ee7bae7d36a12a7e66ac7b610f75b1fa01 - reformating entire code for #191

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/310/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
